### PR TITLE
Add Tutorial Development with Clang-Repl(GSoC'23 blog)

### PR DESCRIPTION
**Pull request by Krishna-13-cyber**

### CODE REVIEW

File: content/posts/2023-10-5-Tutorial-Development-with-Clang-Repl.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+author: "Krishna Narayanan"
+date: "2023-10-05"
+tags: ["Clang", "Clang-repl", "Tutorial", "GSoC", "Compiler Research"]
+title: "Tutorial Development with Clang-Repl"
+---
+
+## Introduction
+This project aims to develop tutorials demostrating the current capabilities of
```
Comment (lattner) [inline content/posts/2023-10-5-Tutorial-Development-with-Clang-Repl.md:9]: typo demostrating

File: content/posts/2023-10-5-Tutorial-Development-with-Clang-Repl.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+author: "Krishna Narayanan"
+date: "2023-10-05"
+tags: ["Clang", "Clang-repl", "Tutorial", "GSoC", "Compiler Research"]
+title: "Tutorial Development with Clang-Repl"
+---
+
+## Introduction
```
Comment (lattner) [inline content/posts/2023-10-5-Tutorial-Development-with-Clang-Repl.md:8]: I'd suggest starting with a couple lines saying "I'm Krishna, this blog post is talking about my GSoC project, etc". Folks may not know what "This project" is

**Review by lattner — APPROVED** [review]
This is awesome, thank you for writing this up.  Please ping me when you're ready to merge this, and fyi @vgvassilev

**Comment by vgvassilev** [comment]
Thanks for writing the this blog post Krishna. LGTM when the comments are addressed.

**Comment by Krishna-13-cyber** [comment]
> This is awesome, thank you for writing this up. Please ping me when you're ready to merge this, and fyi @vgvassilev

Hi @lattner,
I have made the required changes, its ready to merge!
Thanks.

**Comment by lattner** [comment]
Oops, I'm sorry I missed this with the holidays etc.  Merging now, thank you again for writing this!

**Pull request merged**